content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -245,6 +245,14 @@ sets this label on that Pod. The value of the label is the name of the Pod being
 See [Pod Name Label](/docs/concepts/workloads/controllers/statefulset/#pod-name-label) in the
 StatefulSet topic for more details.
 
+### scheduler.alpha.kubernetes.io/node-selector {#schedulerkubernetesnode-selector}
+
+Example: `scheduler.alpha.kubernetes.io/node-selector: name-of-node-selector`
+
+Used on: Pod
```

Comment:
Is this used on Pods or on namespaces?

Reply:
@chetak123, did you have an answer for that question?

Reply:
Hello @sftim (very sorry for the late reply) actually I'm not available on my system so can't test it out locally, but after reading about it since admission controller is applied on a namespace, the `Used on`: parameter should be `Namespace`.